src/Components/GenreSelector/GenreSelector.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+const title = 'Genre Selector';
+
+export class GenreSelector extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            selected: parseInt(props.selectedGenre),
+            genreList: props.genreList
+        };
+    }
+
+    selectGenre(genre){
+        let state = {...genre};
+        state.selected = genre.id;
+        this.setState(state);
+        this.props.onSelectGenre(genre)
+    }
+
+    render() {
+        return (
+            <div className='genreSelectContainer'>
+                <h2>{title}</h2>
+                <div>
+                    {this.state.genreList.map(
+                        (genre) => <span 
+                        className={`${genre.id === this.state.selected ? "active" : ""}`} 
```

Comment:
You can check that library:
https://www.npmjs.com/package/classnames

it's very popular and really useful in case of bit classes and classes under conditions